models/ed/R/write.configs.ed.R

```diff
@@ -163,10 +165,28 @@ write.config.ED2 <- function(trait.values, settings, run.id, defaults = settings
     ed2in.text <- gsub("@PHENOL_START@", "", ed2in.text)
     ed2in.text <- gsub("@PHENOL_END@", "", ed2in.text)
   }
-  
+
+  ## -------------
+  # Special parameters for SDA
+  # 
+  # TODO: This needs to be whatver line in pecan.xml that indicates that SDA 
+  # will be run.
+  if (!is.null(settings$sda)) {
+    # Default values
+    sda_tags <- list(isoutput = 3,    # Save history state file
+                     unitstate = 1,   # History state frequency is days
+                     frqstate = 1)    # Write history file every 1 day
```

Comment:
Why would you be writing history every 1 day? Shouldn't you be using the assimilation frequency setting?